### ISSUE TITLE
Per child permissions for users

### ISSUE THREAD

**Issue by rplankenhorn** [babybuddy/babybuddy#187]
It would be great if you could give certain users access to a limited set of children.  For example, my brother and sister in law are expecting a child and it would be much easier if I could just add them as a user to my current Baby buddy server and then just lock it down for them to only be able to update things for their child.

Another possibility would be to just allow users to not show children for them in any child drop downs.

**Comment by cdubz** [comment]
Yes, this would be really nice to have. I think about it on occasion but never commit to it. There are a number of resources about object level permissions in Django but I have never found anything I would call “definitive”. I’m happy to look in to this but not sure when I will have time. PRs welcome as always (:

**Comment by rplankenhorn** [comment]
@cdubz Maybe an easier implementation would be if you could "hide" children from your account.  It would be almost the same functionality, just not enforced.

**Comment by cdubz** [comment]
Eh, I don't really see any reason _not_ to use an object permissions approach. It's just a question of what is the best/most maintainable way to do it. I think there are good options available that just need to be researched for this use case. "Hiding" feels a little like it may even be more effort to introduce (and maintain).

**Comment by Jdwoodson** [comment]
Just wanted to comment and throw a +1 on this enhancement.
I think this would be incredibly beneficial and a huge game changer for small home daycare providers or like @rplankenhorn mentioned being able to share my deployment with other friends/family.
